package.json

```diff
@@ -23,25 +23,26 @@
     "gulp": "*"
   },
   "dependencies": {
+    "babel-eslint": "^7.1.1",
     "del": "^2.0.2",
-    "godaddy-style": "^3.0.0",
-    "gulp-eslint": "^2.0.0",
+    "eslint-plugin-json": "^1.2.0",
+    "eslint-plugin-mocha": "^4.8.0",
+    "eslint-plugin-react": "^6.10.0",
+    "godaddy-style": "^4.0.1",
```

Comment:
Why `godaddy-style` here? I would suggest using the `eslint-config-godaddy*` packages:

- `eslint-config-godaddy`
- `eslint-config-godaddy-react`
- `eslint-config-godaddy-es5`

unless you're trying to support _all_ of those in which case it might be useful to still depend on `godaddy-style`